crates/fj-viewer/src/input/rotation.rs

```diff
@@ -26,21 +26,24 @@ impl Rotation {
 
     pub fn apply(&self, diff_x: f64, diff_y: f64, camera: &mut Camera) {
         if self.active {
-            let rotate_around =
-                self.focus_point.0.unwrap_or_else(Point::origin);
+            let rotate_around: Vector<3> =
+                self.focus_point.0.unwrap_or_else(Point::origin).coords;
 
             let f = 0.005;
 
             let angle_x = diff_y * f;
             let angle_y = diff_x * f;
 
-            let trans = Translation::from(rotate_around);
+            let trans = Transform::translation(rotate_around);
```

Comment:
Again, this is nitpicky, but we're rotation around a point, not a vector. So maybe leave `rotate_around` as a `Point`, and add the `.coords` here, where a vector is needed:
```suggestion
            let trans = Transform::translation(rotate_around.coords);
```